### PR TITLE
Implemnted PE703

### PR DESCRIPTION
**Pull request by nikita-pardeshi-github**
Hi Team, I have implemented PE703, please review it.

### CODE REVIEW

**Review by Chris33871 — COMMENTED** [review]
Hi @nikita-pardeshi-github 

It looks good on my end 👍 
PR 141

##
Not too sure but I think you might also need to update the GraphQL-README.md file.

**Comment by nikita-pardeshi-github** [comment]
> Hi @nikita-pardeshi-github
> 
> It looks good on my end 👍 PR 141
> 
> Not too sure but I think you might also need to update the GraphQL-README.md file.

Thanks @Chris33871 for reviewing it. I will update GraphQL readme with newly added field semiautomatic_harassment.

File: brand/admin.py [hunk 1]
```diff
@@ -349,6 +352,19 @@ def num_linked(self, obj):
...
+        This function is to create object of commentary model when default values are
+        provided by user in admin portal and save the data in respective database.
+        """
+        super().save_model(request, obj, form, change)
+        try:
+            obj.commentary
+        except ObjectDoesNotExist as e:
+            commentary_obj = Commentary.objects.create(brand_id=obj.id)
+            obj.commentary = commentary_obj
+            obj.save()
```
Comment (brugo-dev) [inline brand/admin.py:366]: Curious about why you need to add this.  Was the form not saving commentary properly?

Comment (nikita-pardeshi-github) [reply brand/admin.py:366]: Yes, commentary object wouldn't get created when user enters all default values. So I added this logic.

**Comment by PooriaT** [comment]
Hi @nikita-pardeshi-github ,

I checked it and it seems that it is okay.Thanks.

**Review by PooriaT — APPROVED** [review]
Hi @nikita-pardeshi-github ,

I checked it and it seems that it is okay. Thanks.

**Pull request merged**